packages/thumbprint-atomic/generate-css.js

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
+
+const fse = require('fs-extra');
+const sass = require('sass');
+const postcss = require('postcss');
+const autoprefixer = require('autoprefixer');
+const cssnano = require('cssnano');
+const nodeSassImporter = require('./node-sass-importer');
+
+/**
+ * Ensures that errors throw an error with a stacktrace.
+ */
+process.on('unhandledRejection', error => {
+    throw error;
+});
+
+const compileSass = async fromFile => {
+    const { css } = sass.renderSync({
+        file: fromFile,
+        importer: nodeSassImporter,
+    });
+
+    const processedCss = await postcss([autoprefixer, cssnano]).process(css, {
+        from: undefined,
+    });
+
+    const outputFileName = fromFile.replace('src/', 'dist/').replace('.scss', '.css');
+
+    return fse.outputFile(outputFileName, processedCss);
+};
+
+(async () => {
```

Comment:
Could we add a comment here explaining why we need the file and can't use the CLI directly?

Reply:
Done